src/Workleap.Extensions.Mongo/Indexing/IndexRegistry.cs

```diff
@@ -12,7 +12,7 @@ public IndexRegistry(IEnumerable<Type> documentTypes)
     {
         foreach (var documentType in documentTypes)
         {
-            if (!MongoReflectionCache.IsConcreteMongoDocumentType(documentType))
+            if (!documentType.IsConcreteMongoDocumentType())
```

Comment:
Cleaner that way! :) It was indeed a bit weird calling a static method in MongoReflectionCache!